vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -209,18 +209,21 @@ See the [Plugin Environment Variables](../../tutorials/devops/plugin-environment
   ]
 ```
 For more information, go to the [API Management](../getting-started/ent-api.md) page.
+
 ### Command Specification
-|Name|Type|Required|Description|
-|:-|:-|:-|:------------------------|
-|`build`|String|No|Custom build command|
-|`run`|String|No|Custom run command|
+|Name|Type|Required| Default Values| Description|
```

Comment:
All the other tables have 'possible values' but this column is Default but it worked nicer than a note. Unsure if readers will notice this header is different. Can change to note ...

Reply:
"default values" is not accurate unless both values are used. if it's one or the other it should be "possible values"

Reply:
i guess i don't understand; a default value is what is used without user modification. if that's the case here then it's correct, but if only one of the two is used depending on the stack, then having "default value" as the header with a note to that effect is appropriate

Reply:
I was unsure which was more clear, since it doesn't follow the same format as all the other tables on the page. Since it created confusion for you anyways, think it makes more sense to move it below.